zc_events/client.py

```diff
@@ -238,11 +240,40 @@ def send_email(self, *args, **kwargs):
             with attachments {} and files {}'''
             logger.info(msg.format(email_uuid, to, from_email, attachments, files))
 
-        event_data = generate_email_data(*args, **kwargs)
+        event_data = generate_email_data(email_uuid, *args, **kwargs)
 
         if logger:
             logger.info('MICROSERVICE_SEND_EMAIL: Sent email with UUID {} and data {}'.format(
                 email_uuid, event_data
             ))
 
         self.emit_microservice_event('send_email', **event_data)
+
+    def emit_index_rebuild_event(self, event_name, resource_type, model, batch_size, serializer, queryset=None):
```

Comment:
IMO, optimizing for a case like this should not be a must, at least for the moment. The largest table I was able to come around is `ScheduledMeal` (in legacy) with over 400k records. The speed is fairly well. And, given that we will rarely need to rebuild index, probably not worth additional hours of coding & testing.